frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -77,17 +81,31 @@ class SlideInDialog
       {
         val errorInfo = git4idea.validators.checkRefName(it.text)
         if (errorInfo != null) error(errorInfo.message)
-        else if (it.text == parentName)
+        else if (it.text == parentName) {
```

Comment:
I'd give some meaningful name to `it`, it's used in too many too spread-out places to be just called `it`